docs/source/cn/random_ops.py

```diff
@@ -0,0 +1,164 @@
+import oneflow
+from docreset import reset_docstr
+
+reset_docstr(
+    oneflow.rand,
+    r"""rand(*size, out=None, generator=None, dtype=None, layout=None, device=None, placement=None, sbp=None, requires_grad=False) -> Tensor
+
+    返回一个由在区间 [0, 1) 上均匀分布的随机数填充的新 tensor 。
+
+    输出 tensor 的形状由变量 :attr:`size` 决定。
+
+    参数：
+        - **size** (int... 或 oneflow.Size): 定义输出张量的形状。可以是数字变量，或者集合例如列表或元组，或者 oneflow.Size 
+        - **out** (可选): 输出张量
+        - **dtype** (flow.dtype, 可选): 返回张量的数据类型。默认： ``flow.float32`` 
+        - **layout** (可选): 返回的 Tensor 的布局。
```

Comment:
```suggestion
        - **layout** (可选): 返回的 Tensor 的 layout。
```
layout OneFlow 应该还没支持的，暂时不要翻译。因为直译成布局也不好理解，以后可能真决定翻译时可能要讨论个好理解的专有名词。